tools/e133/ManagementEndpoint.h

```diff
@@ -30,7 +30,7 @@ using ola::rdm::RDMResponse;
 using ola::rdm::RDMCallback;
 
 /**
- * The ManagementEndpoint handles RDMCommands directed at this E1.33 device. It
+ * The ManagementEndpoint handles RDMCommands directed at this E1.33 Component. It
```

Comment:
capital C is on purpose?

Reply:
Yeah, I think from the spec that both Controllers and Devices now have management endpoints, I believe Component (or RPT Component) is the overarching term for Controller and/or Device.

Reply:
"thing" would also be acceptable! ;)

Reply:
I rather meant that the previous "device" was lower-case and the new "Component" is upper-case. No issue with the wording itself

Reply:
I think that was device in the generic sense, whereas I was capitalising the official term from the standard to match the standard.